Update readme (#8)
kyma-incubator/terraform-provider-gardener: examples/azure/README.md

> # Gardener Azure Example

Reviewer: ```suggestion
# Gardener Azure
```

Author: applied change

> This is a configuration example to deploy an azure cluster using gardener. You need to make sure that the secret binding parameter is "azure_secret_binding" for the deployment to work properly and that the resource is "gardener_azure_shoot".

Reviewer: ```suggestion
Use this configuration example to deploy an Azure cluster using Gardener.
 >**NOTE**: For the example to work properly, make sure that the **secret_binding** parameter is set to `azure_secret_binding` and that the **resource** is `gardener_azure_shoot`.
```

Author: applied change

> For now the following cidrs are obtrained from the similar clusters deployed on gardener, in the future these values should be obtained programatically.

Reviewer: ```suggestion
Currently, values for the following CIDRs are obtained from the similar clusters deployed on Gardener:
```

Author: applied change

```bash
   vnetcidr          = "10.250.0.0/16"
  workercidr        = "10.250.0.0/22"
```

> multiple workers can be defined as follows

Reviewer: ```suggestion
You can also define multiple workers:
```
